tox.ini

```diff
@@ -9,7 +9,8 @@ envlist = py27,py34,py35,py36,py37,py38,py39,{py2,py3}-nosasltls,doc,py3-trace,c
 minver = 1.8
 
 [testenv]
-deps = coverage
+# coverag 5.0.2 is failing with "sqlite3.OperationalError: disk I/O error"
+deps = coverage < 5.0
```

Comment:
Can this be removed now? Would `coverage != 5.0` have been better?

Reply:
Last time I tried the whole 5.x series of coverage was broken. There is some issue with sqlite.

Reply:
Let's see how the other PR ends up.